libgadget/run.c

```diff
@@ -500,7 +500,7 @@ set_units(void)
     init_cosmology(&All.CP, All.TimeIC);
 
     if(All.InitGasTemp < 0)
-        All.InitGasTemp = DMAX(All.MinGasTemp, All.CP.CMBTemperature / All.TimeInit);
+        All.InitGasTemp = All.CP.CMBTemperature / All.TimeInit;
```

Comment:
One concern was at high z this would be too low and cause over-cooling problems to surface.

Reply:
Quite so! I'll revert the change.

Reply:
This is now applied in init.c